charts/testkube-api/templates/deployment.yaml

```diff
@@ -90,6 +90,12 @@ spec:
               {{- else }}
               value:  "testkube-minio-service-{{ .Release.Namespace }}:{{ .Values.storage.endpoint_port }}"
               {{- end }}
+            - name: "STORAGE_BUCKET"
+            {{- if .Values.storage.bucket }}
+              value:  "{{ .Values.storage.bucket }}"
+            {{- else }}
+              value:  "testkube-{{ uuidv4 }}"
```

Comment:
will it be the same for each update?

Reply:
Removed the uuid, user will have to take care for AWS S3 to provide a unique name

Reply:
Might be to use "testkube-{{ .Release.Namespace }"  by default?